config.yaml

```diff
@@ -8,10 +8,13 @@ stream_options:
   clob_pair_ids:  # IDs of the markets to stream
     - 0
     - 1
+  subaccount_ids:
+    - dydx1nzuttarf5k2j0nug5yzhr6p74t9avehn9hlh8m/0
```

Comment:
cp the `        # parse subaccount ids, each subaccount id is string with format owner/number` comment here?

Reply:
done